account_menu_invoice_refund/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
```

Comment:
Why do we need this file in i18n?

Reply:
This file is in root folder of the module, not in i18n, and is needed by default for python logic.

Reply:
O! I'm so sorry, lost my sight